### ISSUE TITLE
Collection view not functioning as documented; not returning rows

### ISSUE THREAD

**Issue by njwfish** [arturtamborski/notion-py#55]
jamalex's codebase does return the rows correctly here but obviously I'm trying to pivot to a supported repo.

Not sure whats going on here; its possible I have screwed up pulling the collection view? I cannot find a `Collection` object associated with my view.

```
>>> cv.get_rows()
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'TableView' object has no attribute 'get_rows'
>>> cv.collection.get_rows()
[
  <CollectionViewBlock (id='07...d1', caption='', type='page', collection=None, title='')>,
  <CollectionViewBlock (id='23...a5', caption='', type='page', collection=None, title='')>
]
```

**Comment by arturtamborski** [comment]
Hi!

TLDR: The imports have changed (and also I broke the library a bit...).

Longer explanation:
The collections should work just as they did in original repo but the respective classes are now in completely different places because I rewrote the library a bit in order to have a better structured class layout (instead of the original one-file-fits-all approach).

This unfortunately will break existing code (which I think I should point out more clearly in readme).
But still, the collections and `get_rows()` should work just as before.

The basic distinction between these two is as follows:
```
     original repo               -              my repo
notion.collection.Collection     -> notion.block.collection.basic.CollectionBlock
notion.collection.CollectionView -> notion.block.collection.view.CollectionView
notion.collection.TableView      -> notion.block.collection.view.TableView
```

Other thing to note is that the [`TableView`](https://github.com/jamalex/notion-py/blob/master/notion/collection.py#L235) from the original repo did not provide the `get_rows()` method as well.
This can be seen [here](https://github.com/jamalex/notion-py/blob/master/notion/collection.py#L199), this class (as well as its super) does not have `get_rows()`.

I wanted to finish that comment here and go on with other stuff but just out of curiosity I wanted to check what exactly is going on here and why your `get_rows()` returns `CollectionViewBlock` instead of `CollectionRowBlock` as it should.

Turns out, I've introduced a bug! :) Will fix that asap but for now that shorty explanation is all I have until I find the bug.

**Comment by arturtamborski** [comment]
Update - I found and fixed it here https://github.com/arturtamborski/notion-py/commit/94f7f31dbe88a825b1bb2a209b7fdaea0a1ec1ed#diff-9723d9e23742b7a2da27e9cb92a6f025R95
Will do the release later today.

It should now work just fine:

Example code:
```python3
import os
from notion.client import NotionClient

client = NotionClient(os.environ["NOTION_TOKEN_V2"])
page = client.get_block(os.environ["NOTION_PAGE_URL"])

cvb = page.children[1]
print(cvb.collection.get_rows())
```

Example output:
```
[
  <CollectionRowBlock (id='8fbfe98b-2560-45b7-8360-33ab58b7feca', type='page', title='test3')>,
  <CollectionRowBlock (id='84d7ff5e-e256-4ad4-b32a-25467147dbb6', type='page', title='test2')>,
  <CollectionRowBlock (id='88a1988a-f522-4949-85c8-9e1345759bbd', type='page', title='test1')>,
]
```

Example page:
![image](https://user-images.githubusercontent.com/8676740/94131226-f4299180-fe5d-11ea-8680-5a7dfccdf44b.png)

**Comment by arturtamborski** [comment]
This fix is now avaiable in 0.0.5 - https://pypi.org/project/notion-py/
